lib/pubid/iso/parser.rb

```diff
@@ -40,7 +42,10 @@ class Parser < Parslet::Parser
     #       #   "pas" / "r" / "tr" / "ts" / "tta"
     rule(:type) do
       (str("DATA") | str("ISP") | str("IWA") | str("R") | str("TTA") |
-        str("TS") | str("TR") | str("PAS") | str("Guide") | str("GUIDE")).as(:type)
+        str("TS") | str("TR") | str("PAS") | str("Guide") | str("GUIDE") |
+        # Russian types
+        str("ТС") | str("ТО")
```

Comment:
Are there Russian names for PAS, IWA, ISP, TTA and R (R stands for recommendation)?